core/java/iesi-core/src/main/java/io/metadew/iesi/connection/HostConnection.java

```diff
@@ -266,11 +266,11 @@ public ShellCommandResult executeRemoteCommandExec(String shellPath, String shel
                     HostConnection hostConnection = null;
                     if (i < jumphostConnections.length) {
                         jumphostConnection = jumphostConnections[i];
+                        String finalJumphostConnection = jumphostConnection;
```

Comment:
Why assigning jumpHostConnection to finalJumpHostConnection ? I see in next lines that jumpHostConnection doesn't change, nothing change between jumpHostconnection and finalJumpHostConnectio. It's not redundant ?

Reply:
this is a requirement to use it in the stream on line 272

Reply:
Oh okay I see